postcode_lookup/locale/cy/LC_MESSAGES/messages.po

```diff
@@ -339,6 +353,64 @@ msgstr ""
 msgid "Opening times"
 msgstr "Oriau agor"
 
+#: postcode_lookup/templates/includes/postal_votes.html:3
+#: postcode_lookup/templates/includes/postal_votes.html:14
+msgid "Apply for a postal vote"
+msgstr ""
+
+#: postcode_lookup/templates/includes/postal_votes.html:5
+msgid ""
+"If you know that you won't be able to get to the polling station on "
+"polling day, you can vote by post."
+msgstr "Os byddwch yn gwybod na fydd modd i chi gyrraedd eich gorsaf "
+"bleidleisio ar y diwrnod pleidleisio, gallwch bleidleisio drwy'r post."
```

Comment:
We already had translations for some of the text.
Also there were some bits where I pinched a bit of text from
https://www.electoralcommission.org.uk/voting-and-elections/ways-vote/apply-vote-post
and then took the corresponding translation from
https://www.electoralcommission.org.uk/cy/pleidleisio-ac-etholiadau/ffyrdd-i-bleidleisio/gwneud-cais-i-bleidleisio-drwyr-post
but there are some bits where we are missing a translation and will need to get them.